File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int i=0, j=numbers.size()-1;
        vector<int> output;
        int num; 
        while(i<j){
            num=numbers[i]+numbers[j];
            if(num==target){
                output.push_back(i+1);
                output.push_back(j+1);
                return output;
            } else if(num<target){
                i++;

            }else{
                j--;
            }
        }
        return output;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        output=[]
        for i in range(0,len(nums)-1):
            for j in range(0,len(nums)-i-1):
                if (nums[i]+nums[j+i+1])==target:
                    output.append(i)
                    output.append(j+i+1)
                    return output